packages/fxa-content-server/app/styles/modules/_graphic.scss

```diff
@@ -45,6 +45,14 @@
   }
 }
 
+.graphic-connect-another-device-qr-code {
+  background-image: image-url('qr_code_firefox_mobile.svg');
```

Comment:
Not relevant to this _exact_ line, but I can't seem to add a comment to the SVG file above&hellip;
IIRC, we don't support dark mode on FxA, but do we want to proactively add a white background to the QR code SVG.

Reply:
That's an interesting one. Sounds like something it would be useful to have a policy on